README.md

```diff
@@ -140,34 +141,34 @@ _Note: Attribute keys starting with `ATL.` require the [Active Token Effects](ht
+| `Eldritch Adept`            | _Feat_                                | `Schauerlicher Adept`          | `Adepte occulte`                                                            | `Adepto sobrenatural`             | `Adepto místico`                                  |
+| `Eldritch Invocation`       | _Warlock feat_                        | `Schauerliche Anrufung`        | `Invocation occulte` / `Manifestation occulte`                              | `Invocación sobrenatural`         | `Invocação mística`                               |
+| `Ethereal Sight`            | _Monster feat_                        | `Ätherische Sicht`             | `Vision éthérée` / `Vue éthérée`                                            | `Visión etérea`                   | `Visão etérea`                                    |
+| `Fairy`                     | _Race_                                | `Fee`                          | `Fée`                                                                       | `Hada`                            | `Fada`                                            |
+| `Ghostly Gaze`              | _Warlock invocation_                  | `Geisterhafter Blick`          | `Regard fantomatique`                                                       | `Mirada fantasmal`                | `Olhar fantasmagórico`                            |
+| `Hexblood`                  | _Race_                                | `Hexblut`                      | `Sang maudit`                                                               | `Sangre maléfica`                 | `Sangre maléfica`                                 |
+| `Hollow One`                | _Supernatural Gift_                   | `Leerwandler`                  | `Celui-qui-est-creux`                                                       | `Aquel que está vacío`            | `Oco`                                             |
+| `Magic Awareness`           | _Barbarian (Path of Wild Magic) feat_ | `Magische Wahrnehmung`         | `Conscience magique`                                                        | `Conciencia mágica`               | `Percepção mágica` / `Vestígios mágicos`          |
+| `Satyr`                     | _Race_                                | `Satyr`                        | `Satyre`                                                                    | `Sátiro`                          | `Sátiro`                                          |
+| `See Invisibility`          | _Spell_                               | `Unsichtbares sehen`           | `Détection de l'invisibilité`                                               | `Ver invisibilidad`               | `Ver invisibilidade` / `Ver o invisível`          |
+| `Sense Magic`               | _Monster feat_                        | `Magie spüren`                 | `Détection de la magie` / `Perception de la magie`                          | `Sentir magia`                    | `Sentir magia`                                    |
```

Comment:
Perhaps this translation needs confirmation as I took it from the weird sources.

Reply:
I don't recognize the "Vestígios mágicos" but would be good if it can be there. Besides that, I think you can just use capital on the names. Like "Vestígios mágicos" would be "Vestígios Mágicos".

The rest is perfect.